src/locales/hu.json

```diff
@@ -24,15 +33,75 @@
 		},
 		"limits": {
 			"description": "Kiírja a várólista beküldési korlátait.",
-			"name": "határok"
+			"name": "határok",
+			"values": {
+				"cooldown": {
+					"description": "Az a minimális idő, ameddig minden felhasználónak várnia kell a saját beküldései között.",
+					"name": "Beküldés Cooldown"
+				},
+				"count": {
+					"description": "Az egyes felhasználók által beküldhető anyagok maximális száma.",
+					"name": "Beadványok száma"
+				},
+				"entry-duration-max": {
+					"description": "A beküldött dal maximális időtartama.",
+					"name": "Max. dal hossza"
+				},
+				"entry-duration-min": {
+					"description": "A beküldött dal minimális időtartama.",
+					"name": "Min. dal hossza"
+				},
+				"queue-duration": {
+					"description": "Az a maximális időtartam, ameddig a várólista eltarthat, ha az összes bejegyzést végponttól végéig lejátszaná. A várólista automatikusan bezárul, ha a beküldés miatt a sor túllépi ezt a korlátot.",
+					"name": "Teljes várólista hossza"
+				}
+			},
+			"responses": {
+				"infinite": "végtelen",
+				"none": "semmi",
+				"title": "Sorkorlátok",
+				"use-cooldown-cmd": "A {cooldown} segítségével megtekintheti a lehűlési időt"
+			}
 		},
 		"nowplaying": {
 			"description": "Kiírja az aktuális dalt a sorban (vagy a legjobb tippemet).",
-			"name": "mitjátszik"
+			"name": "mitjátszik",
+			"responses": {
+				"almost-sure": "85%-ban biztos vagyok benne, hogy ez {usermention} beadványa: {url}",
+				"dj-says": "A DJ azt mondta nekem, hogy ez {usermention} beadványa: {url}",
+				"good-chance": "Jó esély van rá, hogy ez {usermention} beadványa: {url}",
+				"i-think": "Szerintem ez {usermention} beadványa: {url}",
+				"likely": "Nagyon valószínű, hogy ez {usermention} beadványa: {url}",
+				"no-queue": "Jelenleg nincs sor beállítva, így semmi sem játszik le.",
+				"no-song": "Valószínűleg most nem játszik semmi. (Ha van, nem hallom)"
```

Comment:
```suggestion
				"almost-sure": "Úgy 85%-ban vagyok benne biztos, hogy ezt {usermention} küldte be: {url}",
				"dj-says": "Megkérdeztem a DJ-t, szerinte ezt {usermention} küldte be: {url}",
				"good-chance": "Jó esély van rá, hogy ez {usermention}-é: {url}",
				"i-think": "Szerintem ezt {usermention} küldte be: {url}",
				"likely": "Nagyon valószínű, hogy ezt {usermention} küldte be: {url}",
				"no-queue": "Jelenleg nincs sor itt beállítva, így semmi sincs a lemezjátszón.",
				"no-song": "Most semmi nem hallható. (Ha mégis, akkor beszélnem kell az orvosommal)"
```

Reply:
![Screenshot 2023-03-09 at 09 13 36](https://user-images.githubusercontent.com/1500092/224084765-fcc19e2e-676a-4d85-bc58-9bf61a9e95c8.png)

> I need to talk to my doctor

I love this! 😆